engine/apps/telegram/tasks.py

```diff
@@ -111,6 +111,19 @@ def send_link_to_channel_message_or_fallback_to_full_incident(
         )
 
 
+@shared_dedicated_queue_retry_task(bind=True, autoretry_for=(Exception,), retry_backoff=True, max_retries=None)
+def send_link_to_channel_message_or_fallback_to_full_incident(
```

Comment:
Can't just delete this straight away since it can cause celery issues. So introducing a new task with the same internals but a new name and "deprecating" this one.